packages/block-editor/src/components/inspector-controls-tabs/README.md

```diff
@@ -0,0 +1 @@
+TBA...
```

Comment:
Lets maybe either add something basic here about the component and experiment or alternatively remove this file before merge.

Reply:
Thanks. I've removed it until we see how the dust settles during this experiment.